### PR TITLE
fix: updated export/import bodyportal

### PR DESCRIPTION
**Pull request by sooinc**
## Overview
It was brought to my attention that the import for BodyPortal was causing an error:
Error: Can't resolve '../../../../gamut/src/BodyPortal' in '/Users/cassiespain/Desktop/cc/Codecademy/node_modules/@codecademy/gamut-labs/dist/experimental/Popover'

I've updated to import BodyPortal from '@codecademy/gamut' and exported in the gamut/src/index file for it to work.

- [x] Related to past JIRA ticket: LX-4014 & EGG-298 (#846) 
- [x] I have run this code to verify it works

### CODE REVIEW

**Review by JoshuaKGoldberg — APPROVED** [review]
Thanks! Sorry I missed this in the review!

File: packages/gamut/src/index.tsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ export * from './AppBar';
 export * from './AppBar/AppBarSection';
 export * from './Badge';
 export * from './Banner';
+export * from './BodyPortal';
```
Comment (JoshuaKGoldberg) [inline packages/gamut/src/index.tsx:10]: We also use a `.snap` file to test that these get updated. You'll need to update that too. 

https://jestjs.io/docs/en/snapshot-testing

https://app.circleci.com/pipelines/github/Codecademy/client-modules/7471/workflows/ba96d792-fe51-4586-b67a-9f544a545be0/jobs/43445

Comment (sooinc) [reply packages/gamut/src/index.tsx:10]: Ah yes, thanks for calling out! It's been added.

**Comment by sooinc** [comment]
> Thanks! Sorry I missed this in the review!

No worries! Sorry I missed as well!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@4.6.1-alpha.817b7b.0<br/>@codecademy/gamut@16.3.1-alpha.817b7b.0<br/>@codecademy/styleguide@17.10.2-alpha.817b7b.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f9884c7801e740b3d2f6b3a--gamut-preview.netlify.app>https://5f9884c7801e740b3d2f6b3a--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f9884c7801e740b3d2f6b3a>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@4.6.1-alpha.91ecf3.0<br/>@codecademy/gamut@16.3.1-alpha.91ecf3.0<br/>@codecademy/styleguide@17.10.2-alpha.91ecf3.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f9885471e4eb30e80146d08--gamut-preview.netlify.app>https://5f9885471e4eb30e80146d08--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f9885471e4eb30e80146d08>Deploy Logs</a></p>

**Pull request merged**